### ISSUE TITLE
Crashes 

### ISSUE THREAD

**Issue by betninja1** [kochlisGit/ProphitBet-Soccer-Bets-Predictor#59]
Hi do i have to make any changes to the updated model ? I was using the last one just fine and redownloaded with the changes you made and doesnt work at all 
Thanks

**Comment by kochlisGit** [comment]
Hello, unfortunately, the newer version is not compatible with the previous version. However, the new version offers more Machine Learning models, as well as more training options. Addtionally, the evaluation part is now improved using the cross-validation technique and there is also the percentile option to manually adjust the output probabilities of the trained models! If you remember the configuration of your previous model, You can also train the same model in this version. If not, you can start a hyperparameter search, using the newer and more simple interface.

**Comment by betninja1** [comment]
Since i am really newbie to all the python stuff, do i have to download anything so the newer version works? any new pip installs ?

**Comment by kochlisGit** [comment]
Do the following:

1. Check that the previous python version is 3.9. If it is, skip this step. If not, uninstall it and install Python 3.9. Make sure once the installation window opens to check _"Add to path"_ option so that _"pip"_ tool becomes available. To check if Python 3.9 is installed, open cmd (command line) and type "python --version". Alternatively, you can check it from the installed apps in the control panel.
2. Just install the required version libraries. Open the requirements.txt file and check that these version much with the ones installed in your systems. To check all your python libraries, just type in cmd: _"pip list"_. If you don't have a specific library (e.g. tensorflow 2.9.1), you can install it by using _pip install tensorflow==2.9.1_

**Comment by betninja1** [comment]
I ll try that now , thank you very much!

**Comment by betninja1** [comment]
Seems i cant install tensorflow and addons from the pip list

C:\Users\msi>pip install scikit-learn==1.4.0
Collecting scikit-learn==1.4.0
  Using cached scikit-learn-1.4.0.tar.gz (7.7 MB)
  Installing build dependencies ... error
  ERROR: Command errored out with exit status 1:
   command: 'c:\users\msi\appdata\local\programs\python\python39-32\python.exe' 'c:\users\msi\appdata\local\programs\python\python39-32\lib\site-packages\pip' install --ignore-installed --no-user --prefix 'C:\Users\msi\AppData\Local\Temp\pip-build-env-1w0gs0zy\overlay' --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools wheel 'Cython>=0.29.33' 'numpy>=1.25' 'scipy>=1.6.0'
       cwd: None

**Comment by betninja1** [comment]
Sadly same as before , i open the  main file
says this and after that something else and closes fast
2024-02-02 14:11:19.174969: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'cudart64_110.dll'; dlerror: cudart64_110.dll not found
2024-02-02 14:11:19.175285: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.

**Comment by betninja1** [comment]
I keep trying but not sure whats wrong :(

**Comment by kochlisGit** [comment]
Open the command line inside the project directory and type python main,.py. Then send me a screenshot of your screen to see the message.

**Comment by betninja1** [comment]
quick question where i find that ?

**Comment by betninja1** [comment]
the command line you said
its the cmd?

**Comment by kochlisGit** [comment]
Yes, it's the cmd.

**Comment by betninja1** [comment]
<img width="482" alt="Screenshot_248" src="https://github.com/kochlisGit/ProphitBet-Soccer-Bets-Predictor/assets/142240119/b05ca711-d906-4a57-b0e4-d7bd2337b947">

**Comment by betninja1** [comment]
i tried the fix you mention in common errrors didnt work

**Comment by kochlisGit** [comment]
You should navigate to the project directory. For example, if you downloaded the files (Prophitbet) in downloads folder, you shoud etype 

`cd Downloads/Prophitbet`

before typing

`python main.py`

**Comment by betninja1** [comment]
done that as well

**Comment by betninja1** [comment]
All ok now ! 
works but the under over  doesnt give any results after i train the models

**Comment by GrizzyGamer** [comment]
![Screenshot 2024-02-03 165928](https://github.com/kochlisGit/ProphitBet-Soccer-Bets-Predictor/assets/153855217/086696ce-0d92-4771-a09b-38c2ea291726)
can you help me with this issue ?

**Comment by kochlisGit** [comment]
This is just a warning! Nothing to worry about. Did you just double click the main.py? If it does't work, follow the previous instructions at the beginning of this issue.

**Comment by GrizzyGamer** [comment]
After running the main.py in the python command prompt, i should double click on main.py in the directory or what do you mean ?

**Comment by GrizzyGamer** [comment]
absl-py                      2.1.0
alembic                      1.13.1
astunparse                   1.6.3
attrs                        23.2.0
cachetools                   5.3.2
certifi                      2024.2.2
cffi                         1.16.0
charset-normalizer           2.0.12
colorama                     0.4.6
colorlog                     6.8.2
contourpy                    1.2.0
cycler                       0.12.1
flatbuffers                  23.5.26
fonttools                    4.47.2
fuzzywuzzy                   0.18.0
gast                         0.4.0
google-auth                  2.27.0
google-auth-oauthlib         1.0.0
google-pasta                 0.2.0
greenlet                     3.0.3
grpcio                       1.60.1
h11                          0.14.0
h5py                         3.10.0
idna                         3.6
imbalanced-learn             0.9.1
jax                          0.4.23
joblib                       1.3.2
keras                        2.12.0
kiwisolver                   1.4.5
libclang                     16.0.6
Mako                         1.3.2
Markdown                     3.5.2
MarkupSafe                   2.1.5
matplotlib                   3.8.2
ml-dtypes                    0.2.0
numpy                        1.24.3
oauthlib                     3.2.2
opt-einsum                   3.3.0
optuna                       3.5.0
outcome                      1.3.0.post0
packaging                    23.2
pandas                       1.4.3
pillow                       10.2.0
pip                          22.3.1
protobuf                     4.23.4
pyasn1                       0.5.1
pyasn1-modules               0.3.0
pycparser                    2.21
pyparsing                    3.1.1
PySocks                      1.7.1
python-dateutil              2.8.2
pytz                         2024.1
PyYAML                       6.0.1
requests                     2.28.0
requests-oauthlib            1.3.1
rsa                          4.9
scikit-learn                 1.4.0
scipy                        1.12.0
seaborn                      0.11.2
selenium                     4.15.2
setuptools                   65.5.0
six                          1.16.0
sniffio                      1.3.0
sortedcontainers             2.4.0
SQLAlchemy                   2.0.25
tensorboard                  2.12.3
tensorboard-data-server      0.7.2
tensorflow                   2.12.1
tensorflow-addons            0.20.0
tensorflow-cpu               2.15.0
tensorflow-estimator         2.12.0
tensorflow-intel             2.12.1
tensorflow-io-gcs-filesystem 0.31.0
termcolor                    2.4.0
threadpoolctl                3.2.0
tqdm                         4.66.1
trio                         0.24.0
trio-websocket               0.11.1
typeguard                    2.13.3
typing_extensions            4.5.0
urllib3                      1.26.18
Werkzeug                     3.0.1
wheel                        0.42.0
wrapt                        1.14.1
wsproto                      1.2.0
xgboost                      2.0.3 

this is my pip list, all packages are installed, still not running after i 2x click on main.py

**Comment by kochlisGit** [comment]
all right. Type `pip install tabulate`

**Comment by GrizzyGamer** [comment]
everything is good now, it is fixed thanks :)
